quarkchain/constants.py

```diff
@@ -26,3 +26,9 @@
+""".strip()
+)
```

Comment:
I will suggest adding a field ROOT_CHAIN_POSW_CONTRACT_BYTECODE_HASH and use the field to check in shard state

Reply:
will do

Reply:
update: I added the bytecode hash to config, but leaves the actual bytecode in the constants